mscore/plugin/api/part.h

```diff
@@ -25,10 +25,32 @@ namespace PluginAPI {
 
 class Part : public Ms::PluginAPI::ScoreElement {
       Q_OBJECT
-      Q_PROPERTY(int                            startTrack          READ startTrack)
-      Q_PROPERTY(int                            endTrack            READ endTrack)
+      Q_PROPERTY(int                            startTrack           READ startTrack)
+      Q_PROPERTY(int                            endTrack             READ endTrack)
       /// The string identifier for the current instrument. \since MuseScore 3.2
-      Q_PROPERTY(QString                        instrumentId        READ instrumentId)
+      Q_PROPERTY(QString                        instrumentId         READ instrumentId)
+      /// TBD \since MuseScore 3.2
+      Q_PROPERTY(int                            harmonyCount         READ harmonyCount)
+      /// TBD \since MuseScore 3.2
+      Q_PROPERTY(bool                           hasDrumStaff         READ hasDrumStaff)
+      /// TBD \since MuseScore 3.2
+      Q_PROPERTY(bool                           hasPitchedStaff      READ hasPitchedStaff)
+      /// TBD \since MuseScore 3.2
+      Q_PROPERTY(bool                           hasTabStaff          READ hasTabStaff)
+      /// TBD \since MuseScore 3.2
+      Q_PROPERTY(int                            lyricCount           READ lyricCount)
+      /// TBD \since MuseScore 3.2
+      Q_PROPERTY(int                            midiChannel          READ midiChannel)
+      /// TBD \since MuseScore 3.2
+      Q_PROPERTY(int                            midiProgram          READ midiProgram)
```

Comment:
same for this?

Reply:
The property docs are almost filled out!

So what precisely is a midiChannel and midiProgram.

I only have an abstract notion of midi as a physical musical instrument interface.

Reply:
Good question. Next one please ;-)

Reply:
midiProgram is one of the 128 different instruments in General MIDI, midiChannel is one of 16, not sure what exactly, 10 is for drums IIRC

Reply:
or 9 (if 0-based)

Reply:
For me the Wikipedia midi entry clarified it some. 

I'll post all my changes now for review.